api/app/views/mno_enterprise/jpi/v1/admin/invoices/_invoice.json.jbuilder

```diff
@@ -1,2 +1,5 @@
 json.extract! invoice, :id, :price, :started_at, :ended_at, :created_at, :updated_at, :paid_at, :slug,
-              :tax_pips_applied, :billing_address
\ No newline at end of file
+              :tax_pips_applied, :billing_address, :total_due, :total_due_remaining
+
+json.organization invoice.organization
```

Comment:
please select the field you want to extract from organization and bills

Reply:
The changes were committed.

Thanks